<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="Library">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>751</width>
      <height>511</height>
     </rect>
    </property>
    <property name="title">
     <string>Library</string>
    </property>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>721</width>
       <height>431</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Login</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Enter your name : </string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QLineEdit" name="lEName"/>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Enter your password :</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="lEPassword"/>
       </item>
       <item row="2" column="0" colspan="3">
        <widget class="QPushButton" name="pbLogin">
         <property name="text">
          <string>Login</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="3">
        <widget class="QPushButton" name="pbStoreData">
         <property name="text">
          <string>Store Data</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="3">
        <widget class="QPushButton" name="pbRestoreData">
         <property name="text">
          <string>Restore Data</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="3">
        <widget class="QPushButton" name="pbRemoveData">
         <property name="text">
          <string>Remove Data </string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Admin</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="4" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Add Bar Code</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Enter the Book Title</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" colspan="2">
        <widget class="QLineEdit" name="lineEditBookTitle"/>
       </item>
       <item row="1" column="2" colspan="2">
        <widget class="QLineEdit" name="lineEditBookAuthor"/>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Enter the Book Author</string>
         </property>
        </widget>
       </item>
       <item row="10" column="3">
        <widget class="QPushButton" name="pushButtonExit_2">
         <property name="text">
          <string>Exit</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QLineEdit" name="lineEditBookCategory"/>
       </item>
       <item row="3" column="0" colspan="3">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Enter the Book Category</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2" colspan="2">
        <widget class="QLineEdit" name="lineEditBookBarcode"/>
       </item>
       <item row="10" column="0">
        <widget class="QPushButton" name="AddBook">
         <property name="text">
          <string>Add Book</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0" colspan="4">
        <widget class="QListWidget" name="listWidgetResult"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>User</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Enter Title : </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit"/>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Enter Filter : </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox"/>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="Search">
         <property name="text">
          <string>Search</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QListWidget" name="listWidget"/>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="pushButtonClear">
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="pushButtonExit">
         <property name="text">
          <string>Exit</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
